### PR TITLE
111 data location for frontend

### PR DESCRIPTION
**Pull request by berggren**
This PR enables message string support in the Plaso frontend task.
Resolves: #111 
Reviewer: @onager

### CODE REVIEW

File: timesketch/lib/tasks.py [hunk 1]
```diff
@@ -32,11 +61,15 @@ def run_plaso(source_file_path, timeline_name, index_name):
     Returns:
         Dictionary with count of processed events.
     """
+    # Try to read the winevt-rc database path from the config file.
+    data_location = get_data_location(
```
Comment (Onager) [inline timesketch/lib/tasks.py]: This is a bit ugly - maybe move the config retrieval to get_data_location

File: timesketch.conf [hunk 2]
```diff
@@ -61,3 +61,7 @@ UPLOAD_FOLDER = u'/tmp'
 # server is running.
 CELERY_BROKER_URL='redis://ip:port',
 CELERY_RESULT_BACKEND='redis://ip:port'
+
+# Path to Plaso WinEvtx message string database.
```
Comment (Onager) [inline timesketch.conf]: I think its better to refer to this as just the "plaso data directory" - there could be more things added in the future.

File: timesketch/lib/tasks.py (hunk 1 above)
Comment (berggren) [inline timesketch/lib/tasks.py]: This was mostly for testing convenience. I have refactored it now.

File: timesketch.conf (hunk 2 above)
Comment (berggren) [inline timesketch.conf]: Done.

**Comment by Onager** [comment]
LGTM

**Pull request merged**